analysis_options.yaml

```diff
@@ -22,8 +22,11 @@ linter:
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
   rules:
+    prefer_const_constructors: false
```

Comment:
@Danilosimoes isn't  good idea disable rule like that, in my opinion the correct way to resolve this issue is fix in the code >> [here](https://dart.dev/tools/linter-rules#prefer_const_constructors) you can see the rule.

and you can use the tips os android studio:
![image](https://github.com/robsonoduarte/u_vocab/assets/797845/1c754c88-1e84-408d-be9c-dd88f8fc1903)

Reply:
Yes I agree, This is the correct way for resolve this problem. 
I'll resolve this problem with your solution and i learn a litlle bit.
Thanks